### PR TITLE
Remove capture view button and automatically add it

### PR DESCRIPTION
**Pull request by eb-so**
[Notion ticket](https://www.notion.so/filledstacks/81c54266521542a5b12f4f6aee572330?v=e12c9d6c418c4905a7501fb78f130d9b&p=1443c6afad63468194ad9cdb4c04af0a)
- [x]  Remove the Capture view button
- [x]  When capturing and other widget, we check if we have a view
    - [x]  If we don't we generate a view `WidgetDescription` and upload that with the other widget
    
-  Add unit tests

### CODE REVIEW

File: packages/testsweets/lib/src/constants/app_constants.dart [hunk 1]
```diff
@@ -1 +1,3 @@
 const double WIDGET_DESCRIPTION_VISUAL_SIZE = 50;
+// should be a unique one to prevent conflict with user views name
+const String CAPTURED_VIEWS_KEY = '%CAPTURED_VIEWS_KEY%';
```
Comment (FilledStacks) [inline packages/testsweets/lib/src/constants/app_constants.dart:3]: Why are there `%` signs in this string?

File: packages/testsweets/lib/src/services/widget_capture_service.dart [hunk 2]
```diff
@@ -9,8 +10,10 @@ class WidgetCaptureService {
 
   final _cloudFunctionsService = locator<CloudFunctionsService>();
 
+  /// Adding views key that we will add all the captured views to it cause
+  /// they doesn't have a viewName(views viewName is empry string)
```
Comment (FilledStacks) [inline packages/testsweets/lib/src/services/widget_capture_service.dart:14]: See comment in slack. We should set the viewName of the description equal to the view's name. We have to stick to our domain language to ensure no confusion when talking about these things.

**Review by FilledStacks — CHANGES_REQUESTED** [review]

File: packages/testsweets/lib/src/constants/app_constants.dart (hunk 1 above)
Comment (eb-so) [reply packages/testsweets/lib/src/constants/app_constants.dart:3]: Cause I wanted to be as special as possible

Comment (FilledStacks) [reply packages/testsweets/lib/src/constants/app_constants.dart:3]: We can leave it like that for now then.

Comment (eb-so) [reply packages/testsweets/lib/src/constants/app_constants.dart]: We don't need it any more we can now search for the view name if it had an empty widget name

**Review by FilledStacks — APPROVED** [review]
Reaaadddyyyy!! Almost ready for second round of feedback! You guys are moving fast.

**Pull request merged**